File: CSharp/HtmlDocCleaner/DocCleanerWalker.cs
using HtmlWalker;
using HtmlWalker.Platform;
using HtmlWalker.Tag.Util;
using System;
using System.Collections.Generic;
using System.Text;

namespace HtmlDocCleaner
{
    public class DocCleanerWalker : FormattedCloneWalker
    {
        public DocCleanerWalker(WalkerPlatform.WalkerDocument document) : base(document)
        {
            Modifiers.Add(new DocCleanerModifier(Output.Factory));
        }
    }
}

File: Java/htmlwalker/src/main/java/htmldoccleaner/DocCleanerWalker.java
package htmldoccleaner;

import htmlwalker.FormattedCloneWalker;
import htmlwalker.ITag;
import htmlwalker.exception.HtmlWalkerException;
import htmlwalker.platform.WalkerDocument;

public class DocCleanerWalker extends FormattedCloneWalker {
    public DocCleanerWalker(WalkerDocument document)
    {
        super(document);
        modifiers.add(new DocCleanerModifier(output.factory()));
    }

    @Override
    protected <T extends ITag> ITag handleTag(T adapter) throws HtmlWalkerException
    {
        var tag = super.handleTag(adapter);
        tag.attributes().clear();
        return tag;
    }

    @Override
    protected <T extends ITag> ITag startTag(T adapter) throws HtmlWalkerException
    {
        var tag = super.startTag(adapter);
        tag.attributes().clear();
        return tag;
    }
}
